broker/grpc/generate_proto.py

```diff
@@ -177,45 +177,66 @@ class received_protobuf : public io::protobuf<T, Typ> {
 args = parser.parse_args()
 
 message_parser = r'^message\s+(\w+)\s+\{'
-io_protobuf_parser = r'\/\*\s*(\w+::\w+\s*,\s*\w+::\w+)\s*\*\/'
+io_protobuf_parser = r'\/\*\s*(\w+::\w+\s*,\s*\w+::\w+)\s*,\s*(\d+)\s*\*\/'
+ignore_message = "/* Ignore */"
 
 one_of_index = 2
+message_save = []
 
 for directory in args.proto_directory:
     proto_files = [f for f in listdir(
         directory) if f[-6:] == ".proto" and isfile(join(directory, f))]
     for file in proto_files:
+        line_counter = 0
+        flag_ignore = False
         with open(join(directory, file)) as proto_file:
             messages = []
             io_protobuf_match = None
             for line in proto_file.readlines():
+                line_counter += 1
                 m = re.match(message_parser, line)
                 if m is not None and io_protobuf_match is not None:
-                    messages.append([m.group(1), io_protobuf_match.group(1)])
+                    messages.append([m.group(1), io_protobuf_match.group(1), io_protobuf_match.group(2)])
                     io_protobuf_match = None
+                    flag_ignore = True
```

Comment:
flag_ignore = False?

Reply:
in 201 : Already found an io_protobuf, similar to found the comment /* Ignore */ .